lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -214,13 +231,37 @@ Tooltip for each button
 =cut
 
 has tooltips => (is => 'ro',
-    isa => 'HashRef[Str]',
-    default => sub {
-        return {
-            'Save and Retry' => marktext('Save the fixes provided and attempt to continue migration'),
-            'Cancel' => marktext('Cancel the <b>migration</b>')
-    }},
-    required => 0);
+    isa => 'Maybe[HashRef[Str]]', required => 0,
+    default => undef,   # Force initializer call
+    initializer => sub {
+        my ( $self, $value, $writer_sub_ref, $attribute_meta ) = @_;
+        for ($value) {
+            die LedgerSMB::I18N::text(q(No button '[_1]' in test '[_2]'),$_,$self->{name})
```

Comment:
Is this error detecting a programming error or providing the user with valuable feedback?

In case of the former, what are the chances of a true user ever seeing it? If the answer to that is "nil" then I don't think we should bother our translators with the need to translate it.